tuf/api/metadata.py

```diff
@@ -702,6 +702,19 @@ def _verify_length(
                 f"expected length {expected_length}"
             )
 
+    @staticmethod
+    def _validate_hashes(hashes: Dict[str, str]) -> None:
+        if not hashes:
+            raise ValueError("Hashes must be a non empty dictionary")
+        for key, value in hashes.items():
+            if not (isinstance(key, str) and isinstance(value, str)):
```

Comment:
on a more careful re-read, this is a bug -- the order of operations is not what you expect here

Reply:
maybe I'm missing something because I don't see the bug?

Reply:
oops I'm the one that needs new glasses, somehow managed to read this without the enclosing parens